client-dashboard/src/components/gifts/GiftForm.tsx

```diff
@@ -98,6 +98,17 @@ const GiftForm = ({
               meta
             })}
           </Field>
+          <Field
+            name='amount'
+            type='number'
+            validate={composeValidators(required(), minValue(1))}
```

Comment:
Gift cannot be out-of-stock? I think it can - ask Chris about it (if yes, minValue should be 0)